kratos/geometries/line_2d_4.h

```diff
@@ -666,15 +666,15 @@ class Line2D4 : public Geometry<TPointType>
      */
     SizeType EdgesNumber() const override
     {
-        return 2;
+        return 3;
```

Comment:
I think this should be 2.

Reply:
Thank you @rubenzorrilla 
What I understand, the line elements have always 2 edges?
How the faces are defined? Is the FacesNumber=0 for all line elements?

Reply:
Well, I think there is no formal definition. If I'm not remembering wrong, in the @KratosMultiphysics/technical-committee we agreed to consider as edges/faces both endpoints of the line.